### ISSUE TITLE
AnimationTreePlayer should be usable on large number of animations

### ISSUE THREAD

**Issue by slapin** [godotengine/godot#2170]
It is very hard to visually edit animations in AnimationTreePlayer when there is over 30 of them.

AnimationTreePlayer should implement (some of) the following to be easily used for any number of animations:
- Node groups - parts of node sets should be groupable with ability to use as single entity (and still edit). Some animatiions use some 'blend pattern' with others, so these should be probably reusable.
- Interoperation between several AnimationTreePlayers -
  several AnimationTreePlayers could be running in parallel on the same set of object with
  standard AnimationTreePlayer blend modes and filters.
- Ability to drive single output pin to multiple input pins in nodes without requirement of copying
  nodes, even if internally they will be copied, as this really de-clutters visual node display
  getting rid of copies.

**Comment by reduz** [comment]
Very needed feature, but will kick to 3.1, we ran out of time for this

**Comment by thankjura** [comment]
It would be good to add the ability to scale the workspace and move multiple nodes as in blender
![peek 2018-02-08 20-07](https://user-images.githubusercontent.com/2144212/35987119-f6e6b5ce-0d0b-11e8-8551-4ac523bdb254.gif)

**Comment by akien-mga** [comment]
`AnimationTreePlayer` is deprecated in Godot 3.1 and will be removed in Godot 3.2. It is removed by `AnimationTree`, which was written from scratch and likely fixes/deprecates this issue.

If the issue is still relevant in some form with `AnimationTree` in Godot 3.1, please open a new issue about it (potentially linking this one if there is relevant discussion to keep track of).
